spyderlib/widgets/externalshell/namespacebrowser.py

```diff
@@ -437,8 +437,7 @@ def collapse(self):
         """Collapse"""
         self.sig_collapse.emit()
 
-    @Slot(list)

```

Comment:
I can explain what happened... the way it was before the `import_data` was being call with a boolean as a first argument instead of the filenames like it should, when I removed the slot, the function was called as it should have